flask_rest_jsonapi/pagination.py

```diff
@@ -29,7 +29,7 @@ def add_pagination_links(data, object_count, querystring, base_url):
     if querystring.pagination.get('size') != '0' and object_count > 1:
         # compute last link
         page_size = int(querystring.pagination.get('size', 0)) or current_app.config['PAGE_SIZE']
-        last_page = int(ceil(object_count / page_size))
+	last_page = int(ceil(1. * object_count / page_size))
```

Comment:
a clean and side-effect free form of this would be using the `float` builtin

```python
last_page = int(ceil(float(object_count) / page_size))
```

also, this is not a error, this is a python2/python3 differences, the special import `__future__`  have a backport division

python3:
```python
int(ceil(321 / 30)) == 11
```

python2 without `__future__`:
```python
int(ceil(321 / 30)) == 10
```

python2 using `__future__`
```python
from __future__ import division
int(ceil(321 / 30)) == 11
```

so I call for a proper fix using `from __future__ import division`